README.md

```diff
@@ -33,108 +34,242 @@ Menu
+
+As shown above, there are several ways to configure the number of simultaneous virtual users k6 will launch. There are also different ways to specify how long those virtual users will be running. For simple tests you can:
+- Set the test duration by the `--duration`/`-d` CLI flag (or the `K6_DURATION` environment variable and the `duration` script/JSON option). For ease of use, `duration` is specified with human readable values like `1h30m10s` - `k6 run --duration 30s script.js`, `k6 cloud -d 15m10s script.js`, `export K6_DURATION=1h`, etc. If set to `0`, k6 wouldn't stop executing the script unless the user manually stops it.
+- Set the total number of script iterations with the `--iterations`/`-i` CLI flag (or the `K6_ITERATIONS` environment variable and the `iterations` script/JSON option). k6 will stop executing the script whenever the **total** number of iterations (i.e. the number of iterations across all VUs) reaches the specified number. So if you have `k6 run --iterations 10 --vus 10 script.js`, then each VU would make only a single iteration.
+- Set both the test duration and the total number of script iterations. In that case, k6 would stop the script execution whenever either one of the above conditions is reached first.
+
+For more complex cases, you can specify execution stages. They are a combination of `duration,target-VUs` pairs that tell k6 the number of VUs you want the script execution to ramp up or ramp down to over which period. You can set them via the `stages` script/JSON option as an array of `{ duration: ..., target: ... }` pairs, or with the `--stage`/`-s` CLI flags and the `K6_STAGE` environment variable via the `duration:target,duration:target...` syntax.
+
+For example, the following options would have k6 linearly ramping up from 5 to 10 VUs over the period of 3 minutes (k6 starts with `vus` number of VUs, or 1 by default), then staying flat at 10 VUs for 5 minutes, then ramping up from 10 to 35 VUs over the next 10 minutes before finally ramping down to 0 VUs for another 90 seconds.
+
+```js
+export let options = {
+    vus: 5,
+    stages: [
+        { duration: "3m", target: 10 },
+        { duration: "5m", target: 10 },
+        { duration: "10m", target: 35 },
+        { duration: "1m30s", target: 0 },
+    ]
 };
 ```
+Alternatively, you can use the CLI flags `--vus 5 --stage 3m:10,5m:10,10m:35,1m30s:0` or set the environment variables `K6_VUS=5 K6_STAGE="3m:10,5m:10,10m:35,1m30s:0"` to achieve the same results.
 
-Save it as `script.js`, then run k6 like this:
+For a complete list of supported k6 options, take a look at the documentation at [https://docs.k6.io/docs/options](https://docs.k6.io/docs/options).
```

Comment:
For a complete list of supported k6 options, refer to the documentation at [https://docs.k6.io/docs/options](https://docs.k6.io/docs/options).